### DISCUSSION TITLE
Lockfile Maintainence PR's not auto-merging

### DISCUSSION THREAD

**Discussion by raspy8766** [renovatebot/renovate#9585] [Request Help]
## What Renovate type, platform and version are you using?
GitHub App Installation

## Goals
* Auto-merge weekly lockfile updates

## Description
We are using circleci for our CI checks, and when the CI checks are completed it fails to auto-merge. I'd expect renovate bot to trigger another run/job after the ci checks pass that then merges the PR. Any thoughts on what I'm missing in the configuration? Do you think CircleCI isn't reporting the status checks appropriately?

## Relevant debug logs

**Build #336829518**
![image](https://user-images.githubusercontent.com/8533429/115068384-0d9c3480-9ea7-11eb-9934-89bdd98c757b.png)

**Last Renovate Run**
![image](https://user-images.githubusercontent.com/8533429/115074468-95d20800-9eae-11eb-91cc-52105115b036.png)

**CI Checks**
_Note: these checks completed about an hour prior to the screenshot above_
![image](https://user-images.githubusercontent.com/8533429/115070058-489f6780-9ea9-11eb-8951-b8f7155590da.png)

**Renovate Config**
```
{
  "extends": [
    ":separateMajorReleases",
    ":combinePatchMinorReleases",
    ":ignoreUnstable",
    ":prImmediately",
    ":semanticPrefixFixDepsChoreOthers",
    ":updateNotScheduled",
    ":ignoreModulesAndTests",
    "helpers:disableTypesNodeMajor",
    "workarounds:all",
    ":automergeDigest",
    ":disableRateLimiting",
    ":enableVulnerabilityAlertsWithLabel(security)"
  ],
  "dependencyDashboard": true,
  "ignoreNpmrcFile": true,
    "hostRules": [
      {
        "hostType": "npm",
        "baseUrl": "https://hcss.jfrog.io/hcss/api/npm/saturn/auth/saturn",
        "encrypted": {
          "username": "12324234",
          "password": "12341234"
        }
      }
  ]
  "lockFileMaintenance": {
    "schedule": ["at 7am on Wednesday"],
    "enabled": true,
    "automerge": true
  },
  "timezone": "US/Pacific",
  "rangeStrategy": "update-lockfile",
  "enabledManagers": ["npm"],
  "rebaseWhen": "behind-base-branch",
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch", "pin", "digest", "bump"],
      "enabled": false
    }
  ]
}
```

**Comment by rarkins** [comment]
Green checks don't trigger automerge as you expected. Branches will be checked for automerge the next time the bot runs.

**Comment by viceice** [comment]
It needs ~ 1 hour and a second run of renovate to automerge. Renovate doesn't respond to status updates.

**Reply by raspy8766** [reply]
Ah okay, that makes sense. Trying to do a POC with renovate bot, and wanted to demonstrate the auto-merge feature, any good tips on how I could trigger another renovate build? Or should I just wait another hour or two to see if it auto-merges

**Reply by rarkins** [reply]
If you set `"dependencyDashboard": true` in your config then there will be an Issue created. Within that issue there's a checkbox at the bottom which lets you request a new run on the repo. Therefore if you tick that after the tests are green it should trigger Renovate to run and automerge that branch.

**Reply by raspy8766** [reply]
Okay, I'll try that right now

**Reply by raspy8766** [reply]
Success!
![image](https://user-images.githubusercontent.com/8533429/115075668-4b518b00-9eb0-11eb-96ac-db9dce4edc24.png)

Thank you both very much for the help and quick responses!
